<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Создаватель квартального отчёта</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="show_data">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>15</y>
      <width>621</width>
      <height>405</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>      Тема      </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Синтезы</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Образцы и плёнки</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Отчёты</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Нанесения</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>305</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QPushButton" name="file_settings_wg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Настройки файлов</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Настройки тем</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>165</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Отчёты</string>
    </property>
    <widget class="QCheckBox" name="open_check_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>открыть</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="make_xl_bot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Сформировать отчёт 
с именами</string>
     </property>
    </widget>
    <widget class="QPushButton" name="make_xl_bot_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Сформировать отчёт
 без имён</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="open_check">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>открыть</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Даты</string>
    </property>
    <widget class="QCheckBox" name="checkBox_date">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="change_date_but">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Персональные даты</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="year">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>25</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2020</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QComboBox" name="kvartal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1 квартал</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 квартал</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 квартал</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 квартал</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>891</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Строка состояния</string>
    </property>
    <widget class="QLabel" name="current_action">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="read_data_but">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Считать данные</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
